src/main/java/org/fcrepo/spec/testsuite/versioning/LdpcvHttpPost.java

```diff
@@ -38,6 +38,9 @@
  */
 public class LdpcvHttpPost extends AbstractVersioningTest {
 
+    private static final String PREFER_MINIMAL_CONTAINER_HEADER = "return=representation; " +
```

Comment:
As you have seen, the difference between the original resource and the memento is the "last-modified" date. Such as:
```
< <http://localhost:8080/rest/testsuitecontainer01152021175807/2bd16409-e048-4bf8-b175-38c2889651d4> <http://fedora.info/definitions/v4/repository#lastModified> "2021-01-15T22:58:17.255455Z"^^<http://www.w3.org/2001/XMLSchema#dateTime> .
---
> <http://localhost:8080/rest/testsuitecontainer01152021175807/2bd16409-e048-4bf8-b175-38c2889651d4> <http://fedora.info/definitions/v4/repository#lastModified> "2021-01-15T22:58:25.322368Z"^^<http://www.w3.org/2001/XMLSchema#dateTime> .
```

I am a bit uncomfortable adding this Fedora-specific header into the API-Test-Suite... but the other options seem to be to post-process the RDF and ignore/remove certain values, like dates.

This works as is, but I will defer to the team if we are comfortable with this solution.